Translations/nl.js

```diff
@@ -0,0 +1,493 @@
+    ,"auditor_point_to_element" : 'Wijzen naar het element'
+    ,"auditor_unsupported_browser" : 'De codefragment functionaliteit wordt niet ondersteund door deze browser'
+    ,"auditor_page" : 'Pagina'
+    ,"auditor_updated_to" : 'HTML_CodeSniffer werd bijgewerkt naar versie'
+    ,"auditor_view_the_changelog" : 'Bekijk de changelog'
+    ,"auditor_success_criterion" : 'Succescriterium'
+    ,"auditor_suggested_techniques" : 'Voorgestelde technieken'
+    ,"auditor_applies_entire_document" : 'Dit is van toepassing op het hele document'
+
+    //1_1_1.js
+    ,"1_1_1_H30.2" : 'Img element is de enige inhoud van de link, maar bevat geen alt tekst. De alt tekst dient het doel van deze link te beschrijven.'
+    ,"1_1_1_H67.1" : 'Img element met een lege alt tekst mag enkel geen of een leeg titel attribuut hebben.'
+    ,"1_1_1_H67.2" : 'Img element werd gemarkeerd om  genegeerd te worden door Hulptechnologie'
+    ,"1_1_1_H37" : 'Img element bevat geen alt attribuut. Gebruik een alt attribuut om een kort tekstalternatief weer te geven.'
+    ,"1_1_1_G94.Image" : 'Zorg ervoor dat het alt attribuut van dit img element hetzelfde doel dient en dezelfde informatie weergeeft als de afbeelding.'
+    ,"1_1_1_H36" : 'Afbeelding in de indienknop heeft geen alt attribuut.  Zorg voor een tekstalternatief dat de functie van de knop beschrijft, gebruik een alt attribuut.'
+    ,"1_1_1_G94.Button" : 'Zorg ervoor dat alt text van de afbeelding in de indienknop het doel van de knop beschrijft.'
+    ,"1_1_1_H24" : 'Area-element in een image map heeft geen alt attribuut. Ieder area element moet een tekstalternatief hebben dat de functie ervan beschrijft.'
+    ,"1_1_1_H24.2" : 'Zorg ervoor dat het tekstalternatief van het area element hetzelfde doel heeft als het deel van de image map afbeelding waarnaar het verwijst.'
+    ,"1_1_1_G73,G74" : 'Als dit beeld niet volledig kan omschreven worden in een kort tekstalternatief, zorg er dan voor dat een langer tekstalternatief beschikbaar is, bijvoorbeeld in de bodytekst of via een link.'
+    ,"1_1_1_H2.EG5" : 'Img element binnen een link mag geen alt tekst gebruiken die de tekstinhoud van de link herhaalt'
+    ,"1_1_1_H2.EG4" : 'Img element binnen een link heeft een lege of ontbrekende alt tekst terwijl een link net ernaast linktekst bevat. Overweeg beide links samen te voegen.'
+    ,"1_1_1_H2.EG3" : 'Img element binnen een link mag geen alt tekst gebruiken die de tekstinhoud van de link ernaast herhaalt.'
+    ,"1_1_1_H53,ARIA6" : 'Object elementen moeten een tekstalternatief bevatten indien alle andere alternatieven uitgeput zijn.'
+    ,"1_1_1_G94,G92.Object,ARIA6" : 'Controleer of een kort (of indien toepasselijk, lang) tekstalternatief beschikbaar is voor niet-tekstuele inhoud en dat die hetzelfde doel dient en dezelfde informatie toont.'
+    ,"1_1_1_H35.3" : 'Applet elementen moeten een tekstalternatief bevatten in de body van het element, voor browsers die het applet element niet ondersteunen.'
```

Comment:
hyphen between applet and element(en); loan words are often connected with a hyphen if they aren't particularly common